src/plugins/bootstrap.js

```diff
@@ -15,18 +15,36 @@ let parseElement = function (element) {
 
 module.exports = {
 
-  filter(blockTypes, acceptable) {
+  filterBlockTypes(blockTypes, acceptable) {
     if (!acceptable) return blockTypes
 
     return blockTypes.filter(type => acceptable.indexOf(type.id) > -1)
   },
 
+  filterBlocks(blocks, blockTypes) {
+    var _this = this
+    var typeWhitelist = blockTypes.map(blockType => blockType.id)
+    var validBlocks   = []
+
+    blocks.forEach(function(block) {
+      if (typeWhitelist.indexOf(block.type) > -1) {
+        block.blocks = _this.filterBlocks(block.blocks, blockTypes)
+        validBlocks  = validBlocks.concat(block)
+      }
+    })
```

Comment:
Instead of `var _this = this`, what if you used a `=>` function or passed `this` in as the second argument of `blocks.forEach` (to set the scope of the callback )?